swarm64_tpc_toolkit/correctness.py

```diff
@@ -70,22 +72,23 @@ def check_correctness(self, stream_id, query_number):
             self.html += f'<p>{msg}</p>'
             return 'Mismatch'
 
-        # order the columns so that results are easier to compare
-        truth = truth.reindex(sorted(truth.columns), axis=1)
-        result = result.reindex(sorted(result.columns), axis=1)
+        def prepare(df):
+            df = df.fillna('').sort_index(axis=1)
```

Comment:
please add some comments here to explain what this achieves.

Reply:
Yes, will do. Basically: strip NaN, sort columns, natsort rows, re-index. Will make it more clear.